.github/ci.yml

```
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

```suggestion
      - name: Get source code
        uses: actions/checkout@v1
```
Just to keep the formation the same and tell the functionality of each step.